src/dashboard/src/app/welcome/page.tsx

```diff
@@ -1,15 +1,44 @@
+'use client';
+
+import { LoadingAnimation } from '@/components';
+import { useAuth } from '@/hooks';
+import { useAppStore } from '@/store';
+import { redirect } from 'next/navigation';
 import styles from './Welcome.module.scss';
 
 export default function Page() {
+  const { isAuthorized } = useAuth();
+  const userinfo = useAppStore((state) => state.userinfo);
+
+  // Need this because after activating a pre-authorized user so that it will redirect to their home page.
+  if (isAuthorized) redirect('/');
+
   return (
     <div className={`dashboardContainer ${styles.container}`}>
       <div className={styles.welcome}>
         <h2>Request Access</h2>
-        <p>Please email <a href="mailto:placeholder@gov.bc.ca">placeholder@gov.bc.ca</a> to request access to your organization&apos;s dashboard</p>
+        <p>
+          Please email <a href="mailto:placeholder@gov.bc.ca">placeholder@gov.bc.ca</a> to request
+          access to your organization&apos;s dashboard
+        </p>
+        {!userinfo && (
```

Comment:
Some new text to indicate what is going on when the user logs in for the first time.